aim/web/api/tags/views.py

```diff
@@ -41,7 +41,7 @@ async def search_tags_by_name_api(q: Optional[str] = '', factory=Depends(object_
             'id': tag.uuid,
             'name': tag.name,
             'color': tag.color,
-            'description' 'run_count': len(tag.runs),
```

Comment:
This looks very much like an existing bug, no?

Reply:
This is very much an existing bug, thanks for pointing it out. 
I've pushed the fix for this in a separate PR, so can I ask you please to re-base with and remove this change?